ui/dashboardApp/layout/signin/index.module.less

```diff
@@ -155,12 +155,12 @@
 }
 
 .container :global {
-  .formAnimation {
+  .form-animation {
     animation: 0.2s @ease-out-circ 0.5s antZoomBigIn;
     animation-fill-mode: both;
     animation-iteration-count: 1;
   }
-  .landingAnimation {
+  .landing-animation {
```

Comment:
These 2 class names are used in:

```
        <AppearAnimate
          className={styles.landing}
          motionName="landingAnimation"
        />
```

Not sure whether this breaks the behavior, need to check.